packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -236,9 +256,14 @@ export async function init({
         });
     }
 
+    // reset app state if requested
+    if (urlParams.get('fiori-tools-iapp-state')?.toLocaleLowerCase() === 'true') {
```

Comment:
this needs to be 
```
urlParams.get('fiori-tools-iapp-state')?.toLocaleLowerCase() !== 'true'
```

Reply:
Thanks, fixed with [1431102](https://github.com/SAP/open-ux-tools/pull/1825/commits/14311028e5452915a514f88b0e9447871923a244).